doc/content/lfr/heterogeneous_single_assembly_3D/Griffin_standalone_LFR.md

```diff
@@ -0,0 +1,277 @@
+
+!listing lfr/heterogeneous_single_assembly_3D/neutronics_standalone/neutronics.i start=[PinIn_CM] end=uniform_refine include-end=False
+
+[Mesh] shows the radial cross section view of the fine and coarse meshes. Note that the coarse mesh on the right is not a separate mesh that Griffin holds, but just a visual representation of coarse element IDs assigned to the fine mesh on the left.
+
+!media lfr/Mesh.jpg
+       style=width:75%
+       id=Mesh
+       caption=Radial cross section view of (Left) fine mesh and (Right) coarse mesh
+
+### Transport systems
+
+For the transport system, `particle` is `neutron` and `equation_type` is `eigenvalue`. The number of energy groups is 9 given by `G` and `VacuumBoundary` and `ReflectingBoundary` are sideset names renamed in `[rename_sidesets]`. Top and bottom surfaces are vacuum and lateral surfaces are reflective. In the sub-block of `[sn]`, the DFEM-SN scheme is specified. Between `monomial` and `L2-Lagrange` families of basis functions supported for discontinuous elements, the `L2Lagrange` type is selected with the first order since it gives more accurate solution than `monomial` in general. The angular quadrature type (`AQtype`) is set to `Gauss-Chebyshev` for having a freedom to choose a different number of angles in the azimuthal direction and in the polar direction in 3D. The number of azimuthal angles (`NAzmthl`) and that of polar angles (`NPolar`) per octant are set to $3$ and $2$, respectively, from a sensitivity study. The anisotropic scattering order (`NA`) is set to 1. `using_array_variable=true` and `collapse_scattering=true` are recommended options for performance reason in general.
+
+!listing lfr/heterogeneous_single_assembly_3D/neutronics_standalone/neutronics.i start=[TransportSystems] end=# ============================================================================== include-end=False
+
+### Executioner
+
+The `SweepUpdate` executioner is used for the CMFD acceleration. `SweepUpdate` is a special Richardson executioner for performing source iteration with a transport sweeper. This source iteration can be accelerated by turning on `cmfd_acceleration` which invokes the CMFD solve where the low order diffusion equation is solved using transport-corrected Diffusion coefficients. `coarse_element_id` is a name of the extra element integer ID assigned by `CoarseMeshExtraElementIDGenerator` in `[Mesh]`. `cmfd_eigen_solver_type` is the eigenvalue solver for CMFD. There are four options: `power`, `arnoldi`, `krylovshur` and `newton`. `newton` is the default which is recommended to stick with. If `newton` does not converge a solution, either `krylovshur` is the second option to go with, or `cmfd_prec_type` can be changed to `lu` from its default value of `boomeramg` for a small problem. Also one can try `cmfd_closure_type=syw` or `cmfd_closure_type=pcmfd` as different ways to correct Diffusion coefficients, but it is recommended to stick with its default value of `traditional_cmfd`.
```

Comment:
`using transport-corrected Diffusion coefficients` is not quite right, should be replace with `with a convection closure term to make the diffusion system and the transport system consistent`.

Reply:
remove `to correct Diffusion coefficients`.

Reply:
Accepted.